jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```diff
@@ -132,7 +132,7 @@ YUI.add('env-switcher', function() {
       // XXX For now we will create a new env with an auto-incrementing
       // number at the end. Users will be able to customize their env names
       // once there is UX for it.
```

Comment:
Update this XXX; it should probably just be a comment now that we use auto increment if we have no name provided, or you could just remove it.